RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -955,7 +881,7 @@ public static File getCompressedAnalysisToolsOverviewFile() {
         if (getCommandLineCall().isOptionSet(option))
```

Comment:
What's your oppinion about moving all the configuration related methods to the new RoddyAppConfig class?
Then Roddy would be much leaner and we'd also have an object which we could pass around instead of accessing the static Roddy singleton

Reply:
I agree. Configuration is not `Roddy`'s responsibility and a dedicated application configuration object would be more suited. Could be a step towards getting rid of the singleton.